src/app/app-platform/ContentAppPlatform.h

```diff
@@ -81,11 +81,13 @@ class DLL_EXPORT ContentAppPlatform
     // add apps to the platform.
     // This will assign the app to an endpoint (if it is not already added) and make it accessible via Matter
     // returns the global endpoint for this app, or 0 if an error occurred
-    //
+    // If a desiredEndpointId is passed (cannot be less that Fixed endpoint count)
```

Comment:
The function seems to explicitly handle the "cannot be less than fixed endpoint count". This seems to be a mismatch in the docs.

Reply:
since I cannot set a truly invalid default I converted to two different methods. The method that takes a desired endpoint now does not allow any endpoint Id less that Fixed endpoint count.